src/snowflake/cli/plugins/object/__init__.py

```diff
@@ -1,11 +0,0 @@
-from snowflake.cli.api.commands.snow_typer import SnowTyper
-from snowflake.cli.plugins.object.commands import app as show_app
-from snowflake.cli.plugins.stage.commands import app as stage_app
-
-app = SnowTyper(
-    name="object",
-    help="Manages Snowflake objects like warehouses and stages",
-)
-
-app.add_typer(stage_app)  # type: ignore
-app.add_typer(show_app)  # type: ignore
```

Comment:
why?

Reply:
I made it consistent with other plugins (spcs / NADE), but I made the choice between `__init__.py` and `commands.py` arbitrarily - no strong opinions